src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -123,6 +123,27 @@ def test_conversion_as_lone_interface(self):
                 # Any extra custom checks to run
                 self.run_custom_checks()
 
+    def test_add_to_nwb(self):
```

Comment:
```suggestion
    def test_metadata_immutability(self):
```

Reply:
or 'test_no_metadata_mutation'

Reply:
I like both.

Reply:
Did something like the second suggestion and added interface.